packages/gantt/backend/sirius-components-gantt/src/main/java/org/eclipse/sirius/components/gantt/renderer/component/TaskDescriptionComponent.java

```diff
@@ -111,4 +122,15 @@ private boolean shouldRender(String targetObjectId, VariableManager childVariabl
         return true;
     }
 
+    private boolean computeCollapsed(Optional<Task> previousTaskOptional) {
+        if (previousTaskOptional.isPresent()) {
```

Comment:
Your pattern is perfectly acceptable, but rather than using `isPresent`/`get` I prefer to use a `map` directly.
``````
private boolean computeCollapsed(Optional<Task> previousTaskOptional) {
        return previousTaskOptional.map(task -> this.props.ganttEvent()
                .filter(ChangeGanttTaskCollapseStateEvent.class::isInstance)
                .map(ChangeGanttTaskCollapseStateEvent.class::cast)
                .filter(event -> event.taskId().equals(task.id()))
                .map(ChangeGanttTaskCollapseStateEvent::collapsed)
                .orElse(task.detail().collapsed())).orElse(false);
    }